src/audio/002_Hyrule_Field_Program.mus

```diff
@@ -0,0 +1,235 @@
+.desc title "Hyrule Field Program"
+.desc author "Koji Kondo"
+.desc type "program"
+.usefont SOUNDFONT_ORCHESTRA
+
+.sequence sequence_start
+
+# io slot 0 = temp, for communicating with loaded seqs. Initially time of day, with 1 = morning
+# io slot 1 = temp, for seq loading
+# io slot 2 = new external state (input from the game)
+# io slot 3 = current external state (0, 1 or 2)
+# io slot 4 = current internal state (0, 1 or 2, same as slot 3)
+# io slot 5 = last random index
+# io slot 6 = which sequence buffer to load into next (either 0 or 1)
+
```

Comment:
I really like the layout of having io slots documented like this. Not in this PR, but it'd be a good goal to have this documented for all the different player/channel io and the other logic programs in this format. Especially since IO interacts with the code-base a lot so it's good to have quick reference to what IO goes where